packages/components/src/internal/components/forms/input/SelectInput.tsx

```diff
@@ -500,14 +502,15 @@ export class SelectInputImpl extends Component<SelectInputProps, State> {
                             addLabelAsterisk,
                             isFormsy: false,
                             required,
-                            labelClass: !allowDisable ? this.props.labelClass : undefined,
+                            labelClass: !allowDisable ? labelClass : undefined,
                             helpTipRenderer,
                         }}
                         showLabel={showLabel}
                         showToggle={allowDisable}
                         isDisabled={isDisabled}
                         toggleProps={{
-                            onClick: this.toggleDisabled,
+                            onClick: toggleDisabledTooltip ? undefined : this.onToggleChange,
```

Comment:
What about lookup fields with fix container `col.lookup?.containerPath` and `App.isAllProductFoldersFilteringEnabled()` ?

Reply:
ah yes, good call. fix added so that we don't disable the bulk form lookup if there is a col.lookup.containerPath defined (this was changed in QueryFormInputs.tsx where toggleDisabledTooltip is set).

Reply:
and `App.isAllProductFoldersFilteringEnabled()` ?

Reply:
ah yes, that applies here as well. I just added the check for that as well. thanks.